hydroflow_plus_kvs_server/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use hydroflow_plus::hydroflow::bytes::Bytes;
+use hydroflow_plus_kvs::my_kvs;
+use regex::Regex;
+
+fn main() {
+    let (send_0, recv_0) = hydroflow_plus::hydroflow::util::unbounded_channel::<_>();
+    let mut _other_flow = hydroflow_plus_kvs::my_example_flow!(
+        recv_0,
+        &send_0,
+        1,
+        {
+            let _hygiene = Regex::new("hygiene").unwrap();
+            "lol"
+        },
+        "lol"
+    );
+
+    let test = hydroflow_plus_kvs::KVSMessage::Get {
+        key: "lol".to_string(),
+    };
+    let _blah: hydroflow_plus_kvs::__staged::KVSMessage = test;
+
+    let (_send, recv) = hydroflow_plus::hydroflow::util::unbounded_channel::<Bytes>();
+    let mut flow = my_kvs!(true, recv);
```

Comment:
I assume the `true` here means debug is on, but i think nothing gets sent to the kvs so it still doesn't print anything when you run this main function.